kubernetes/riot.yml.erb

```diff
@@ -0,0 +1,121 @@
+---
+apiVersion: v1
+data:
+  config.json: |
+    {
+        "default_server_config": {
+            "m.homeserver": {
+                "base_url": "https://matrix.ocf.berkeley.edu",
+                "server_name": "matrix.ocf.berkeley.edu"
+            },
+            "m.identity_server": {
+                "base_url": "https://matrix.ocf.berkeley.edu"
```

Comment:
I don't think we have a Matrix identity server yet, so not sure if this field is accurate. Might have changed with #8 , but I think this should use `https://vector.im` instead

Reply:
Got it, switched that over.